src/api/thegraph/TheGraphApi.ts

```diff
@@ -59,6 +89,8 @@ export class TheGraphApiImpl {
 
     const response = await fetch(url, {
```

Comment:
let's make the fetch options a const somewhere?

Reply:
I'll leave that up to @alfetopito 
GraphApi will change anyway